packages/jahia/migrations/migrate-to-v23.yaml

```
@@ -131,5 +131,9 @@ actions:
       user: root
 
   removeAzurePythonPackages:
-    - cmd[cp, proc]: pip uninstall azure adal
+    - cmd[cp, proc]: |-
+        pip uninstall azure adal -y
+        pip list | awk '/azure-/ {print $1}' | while read pkg; do
+          pip uninstall pkg -y
+        done
```

I guess something like this will be fastest:
```suggestion
        pip list | awk '$1~/^(adal|azure)/ {print $1}' | xargs pip uninstall -y
```

---

Done!